lang/python/2to3/files/2to3.py

```diff
@@ -0,0 +1,5 @@
+#! $(STAGING_DIR_HOSTPKG)/bin/python3
```

Comment:
Make variables cannot be used here. Use `$(SED)` similar to [this](https://github.com/openwrt/packages/blob/e9731ff6b4cb486886c398beb4c1b5c2619b25b3/lang/python/python3-package.mk#L72) in `Host/Install` to modify the shebang.

Reply:
add